code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -107,15 +107,21 @@
 	if(cooldowntime > world.time)
 		to_chat(user, "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>")
 		return
-	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Nar-Sien Hardsuit")
+	var/choice = alert(user,"You study the schematics etched into the forge...",,"Shielded Robe","Flagellant's Robe","Bastard Sword")
 	var/pickedtype
 	switch(choice)
 		if("Shielded Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/cult_shield
 		if("Flagellant's Robe")
 			pickedtype = /obj/item/clothing/suit/hooded/cultrobes/berserker
-		if("Nar-Sien Hardsuit")
-			pickedtype = /obj/item/clothing/suit/space/hardsuit/cult
+		if("Bastard Sword")
+			if(world.time < 18000)
+				pickedtype = /obj/item/twohanded/required/cult_bastard
+			else
+				cooldowntime = 18000 - world.time
```

Comment:
what on earth is going on here, you're taking the current time away from 30 minutes? but then below you immediately set it to 0?

Reply:
I'm basically piggybacking off the Forge's cooldown timer so I can use the getETA function without having to create another getETA function just for this one message. 

Setting it back to 0 makes sure that it doesn't actually lock out people from using the forge when you get rejected for trying to make the sword early.

Reply:
comment that then, on this line ```// just to get a value out of getETA()```